### PR TITLE
Using TracerLogger instead of *Sensor in all instrumentations

### PR DESCRIPTION
**Pull request by willianpc**
This PR replaces the reference for `instana.Sensor` in all instrumentations by the `instana.TracerLogger` interface.

### CODE REVIEW

File: instrumentation/instaamqp091/go.mod [hunk 1]
```diff
@@ -1,17 +1,10 @@
 module github.com/instana/go-sensor/instrumentation/instaamqp091
 
-go 1.18
+go 1.16
 
 require (
-	github.com/instana/go-sensor v1.53.0
+	github.com/instana/go-sensor v1.55.0
 	github.com/opentracing/opentracing-go v1.1.0
```
Comment (sanojsubran) [inline instrumentation/instaamqp091/go.mod:7]:
```suggestion
    github.com/opentracing/opentracing-go v1.2.0
```

Query: Don't we have to update the opentracing package here ?

File: instrumentation/instaamqp091/go.mod [hunk 2]
```diff
@@ -1,17 +1,10 @@
 module github.com/instana/go-sensor/instrumentation/instaamqp091
 
-go 1.18
+go 1.16
```
Comment (sanojsubran) [inline instrumentation/instaamqp091/go.mod:3]: Query: Why did we move back to go 1.16?

**Review by sanojsubran — APPROVED** [review]
LGTM ! Thanks for taking up this tedious task of updating all the libraries !

File: instrumentation/instaamqp091/go.mod (hunk 2 above)
Comment (willianpc) [reply instrumentation/instaamqp091/go.mod:3]: Good catch.
This is a leftover for when I was trying to apply Go workspaces. I will undo the change.

**Pull request merged**